RSyntaxTextArea/src/main/java/org/fife/ui/rsyntaxtextarea/DefaultTokenFactory.java

```diff
@@ -74,13 +69,14 @@ class DefaultTokenFactory implements TokenFactory {
 	 * request is made and no more tokens are available.
 	 */
 	private void augmentTokenList() {
```

Comment:
The change should be tagged with the JIRA

Reply:
Done now